content/quests/events/demevent_rebel_arms_deal.lua

```diff
@@ -43,154 +43,243 @@ QDEF:AddConvo()
     :ConfrontState("CONFRONT", function() return true end)
         :Loc{
             DIALOG_INTRO = [[
-                * [p] You encountered two people arguing.
+                * You would think trying to sell contraband requires you be quiet. Hush Hush. Generally not try to draw attention.
+                * Either the Jake is new at the job or the Professional Rabble Rouser wants {jakes.hisher} manager.
+                jakes:
+                    !right
                 rise:
                     !left
+                    Are you against Democracy itself? We need the weaponry to protect our revolution!
                 jakes:
-                    !right
-                    No! You can't have those weapons unless you pay for them!
+                    Democracy isn't going to pay the customs fees! This is still a capitalist society, and you still need to pay!
                 rise:
-                    But this is for the cause!
+                    !spit
+                    Bah! That's just bougeroise rhetoric.
                 jakes:
-                    Your cause doesn't pay the bills!
-                * Wow, a conflict. Looks like a good time to intervene.
+                    !angryshrug
+                    Bougeroise? I'm just as poor as the lot of you. Should <i>I<\> starve just so your little revolution has a few more playthings?
+                rise:
+                    This isn't a game, {jake}. This is real life, and real sacrifice needs to be made for real change!
+                * You have a sneaking suspicion the next logical step for this conflict would involve someone getting shot.  
             ]],
             OPT_LEAVE = "Leave them figure this out themselves",
             DIALOG_LEAVE = [[
-                * [p] Or not. You do you.
+                * You pace a bit faster.
+                * The last sound you hear of their squabble is the discharge of a lumin blaster.
```

Comment:
Yeah no one actually dies here, so not sure where the blaster would come from.

Reply:
Mostly just to imply violence. They get into a loud argument, and if you don't come to be the responsible adult one of them just says "screw it", pulls out a gun and shoots the other. Although, yeah, death is probably implied here. Might change it to just "fist fight" or something like that.